budget-binder-multiplatform-app/src/desktopMain/resources/budgetbinder.svg

```diff
@@ -0,0 +1,49 @@
+<svg width="300" height="300" viewBox="0 0 300 300" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Sollten wir nicht eigentlich sowas wie das logo in commonMain packen?

Reply:
Android handelt externe Images anders, deswegen `expect/actual`, iOS wahrscheinlich auch

Reply:
Ja macht Sinn. Etwas nervig, dass das dann als External zählt.